src/super_gradients/training/datasets/classification_datasets/cifar.py

```diff
@@ -76,3 +92,17 @@ def __init__(
             target_transform=target_transform,
             download=download,
         )
+
+    def get_dataset_preprocessing_params(self) -> Dict:
```

Comment:
So, I was wondering how to handle the fact that we dont actually use any resizing in validation, but images are all sized (64, 64).
If we just rely on the preprocessing set here, we might fail in practice running on external source images.

Reply:
Maybe the right thing to do, is to check whether there is any resizing, and if not add a resize to 64 processing.

Reply:
As aggreed, added Resize operations to val dataset params instead.
Fixed in 5151cd975529f700b91b4190b44f5231932ce0cf